react/features/toolbox/components/ToolbarButton.web.js

```diff
@@ -148,30 +232,27 @@ class ToolbarButton extends Component {
     }
 
     /**
-     * Renders popup element for toolbar button.
+     * Hides any displayed tooltip.
      *
-     * @param {Array} popups - Array of popup objects.
-     * @returns {Array}
      * @private
+     * @returns {void}
      */
-    _renderPopups(popups: Array<*> = []): Array<*> {
-        return popups.map(popup => {
-            let gravity = 'n';
-
-            if (popup.dataAttrPosition) {
-                gravity = popup.dataAttrPosition;
-            }
-
-            const title = this.props.t(popup.dataAttr, popup.dataInterpolate);
-
-            return (
-                <div
-                    className = { popup.className }
-                    data-popup = { gravity }
-                    id = { popup.id }
-                    key = { popup.id }
-                    title = { title } />
-            );
+    _onMouseOut(): void {
+        this.setState({ showTooltip: false });
+    }
+
+    /**
+     * Hides any displayed tooltip.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onMouseOver(): void {
+        const { button } = this.props;
+
+        this.setState({
+            showTooltip: isButtonEnabled(button.buttonName)
+                && !button.unclickable
```

Comment:
Why not showing the tooltip for unclickable buttons? It can be useful to know what a button does even though it's disabled for some reason.